pkg/elfwriter/elfwriter_test.go

```diff
@@ -265,7 +265,7 @@ func TestWriter_HasLinks(t *testing.T) {
 	inElf, err := elf.Open("testdata/libc.so.6")
 	require.NoError(t, err)
 	t.Cleanup(func() {
-		inElf.Close()
+		defer inElf.Close()
```

Comment:
```suggestion
	inElf.Close()
```

You don't need to `defer` in here.

Reply:
Ok, I see that it's part of idiomatic Go. Thanks. Removed it at one other place in the file too.

Reply:
👍  It's not about being idiomatic here. First of all, there are no other statements in that scope (the scope is the closure here). And `t.Cleanup` actually does the trick of the `defer.` It gets to run after the test exists.

Reply:
Ah, yes. I see what you're saying now. :+1: